Exiled.API/Features/Roles/Scp079Role.cs

```diff
@@ -23,6 +25,11 @@ public class Scp079Role : Role
         /// <inheritdoc/>
         public override Player Owner { get; }
 
+        /// <summary>
+        /// Gets the <see cref="Scp079PlayerScript"/> script for the role.
+        /// </summary>
+        public Scp079PlayerScript Script => script ? script : script = Owner.ReferenceHub.scp079PlayerScript;
```

Comment:
bruh?

Reply:
rider said to do this for these 2 scripts instead of the script ?? (script = ) thing

Reply:
Yeah ok it doesn't make sense